### DISCUSSION TITLE
What is MockLLM?

### DISCUSSION THREAD

**Discussion by shantanuo** [aiplanethub/beyondllm#41] [General]
Interesting project. Why am I getting "LLM is explicitly disabled. Using MockLLM." The response is similar to the one shown by you, but I get that extra line.

**Comment by tarun-aiplanet** [comment]
Hi @shantanuo 

So basically auto_retriever requires an LLM. We haven't defined the LLM to be null for few specific reasons. That's where be default its taking MockLLM as the default name. In the upcoming iteration, we will add the support to integrate LLM within auto_retriever for smooth functioning
